### ISSUE TITLE
Cannot run on Modeler

### ISSUE THREAD

**Issue by maycown** [IBMPredictiveAnalytics/NN_with_Keras#2]
Hi, I'm trying to execute on Modeler 18.0 FP1 but got an error
![image](https://user-images.githubusercontent.com/31932865/30390348-32e139ae-988c-11e7-8a84-355cfcd8ae5a.png)

**Comment by niallmcc** [comment]
Hello, are you seeing this error running the example stream mnist.str?

**Comment by maycown** [comment]
Yes, and I also tried with another simple one using wine dataset. One of my colleagues tried on OSX and got 
![image](https://user-images.githubusercontent.com/31932865/30394160-a4401974-9898-11e7-9f9d-7b289a91be10.png)

**Comment by niallmcc** [comment]
Thanks.  In both installations please can you tell me what you have set up for the setting eas_pyspark_python_path in modelers config/options.cfg?  if using modeler server, this should be the options.cfg file on the server side.

Also can you double check there are not multiple copies of this setting in the options.cfg file.
